### ISSUE TITLE
Window focus / blur misbehaves

### ISSUE THREAD

**Issue by Gozala** [browserhtml/browserhtml#1053]
If you blur the the window mac window controls remain active while they should gray out. Also when you bring focus back an newtab page input field should get selection, but that does not happen as we don't receive event to do so.

Caused by servo/servo#11526

**Comment by Gozala** [comment]
Depends on #9482 I guess.

**Comment by paulrouget** [comment]
Closing in favor of servo/servo#9482 (P3 at the moment)
